CON-5791 Add documentation for reservations/getChannelManagerDetails end point (#737)
MewsSystems/gitbook-connector-api: operations/reservations.md

| `ReservationId` | string | required | Unique identifier of the reservation. |
| `Data` | string | optional | Reservation data for QR code generation. |

> ## Get reservations&#x27; channel manager details

Reviewer: While I agree with removing the apostrophe, I don't agree with removal of "reservation" from the operation name completely. This operation is about getting additional channel manager details _about reservation_. Naming it just "Get channel manager details" implies, without additional context, that you're getting details _about channel manager_.

How about "Get channel manager details for reservation"?

> ## Get reservations&#x27; channel manager details

Author: @MikeAdamsMews , is it ok like this ?

> ## Get reservations&#x27; channel manager details

Reviewer 2: OK, I take the point @jnv, I don't have a strong opinion on this, I just think we should try and make the title a bit simpler. But I would be okay with just dropping the apostrophe and calling it either `Get reservation channel manager details` or `Get channel manager details for reservation`. I think I would choose the first one, as it's one word and a few characters shorter.

> ## Get reservations&#x27; channel manager details

Author: ok then lets keep it as it is 👍

> ### Restricted!
> This operation is currently in beta-test and as such it is subject to change.

> Returns additional channel manager-related details for provided reservations. Currently returns only requested rate codes of the reservations. Note this operation supports [Portfolio Access Tokens](../concepts/multi-property.md).

Reviewer 2: ```suggestion
Returns channel manager-related details for the specified reservations. Currently returns only requested rate codes of the reservations. Note this operation supports [Portfolio Access Tokens](../concepts/multi-property.md).
```

```javascript
{
  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
  "Client": "Sample Client 1.0.0",
  "ReservationIds": [
    "9b59b50d-bd32-4ce5-add8-09ea0e1300e7"
  ]
}
```

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| `ClientToken` | string | required | Token identifying the client application. |
| `AccessToken` | string | required | Access token of the client application. |
| `Client` | string | required | Name and version of the client application. |
| `ReservationIds` | array of string | required, max 100 items | Unique identifiers of the `Reservations`. |

### Response

```javascript
{
  "ChannelManagerDetails": [
    {
      "ReservationId": "9b59b50d-bd32-4ce5-add8-09ea0e1300e7",
      "RequestedRateCode": "TKyFHQwLtEBfs1akKan1a2Ea4"
    }
  ]
}
```

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| `ChannelManagerDetails` | array of [Reservations' channel manager details](reservations.md#reservations-channel-manager-details) | required | List of reservations' channel manager details. |

> #### Reservations&#x27; channel manager details

Reviewer 2: ```suggestion
#### Reservation channel manager details
```

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |

> | `ReservationId` | string | required | Unique identifier of the `Reservations`. |

Reviewer 2: ```suggestion
| `ReservationId` | string | required | Unique identifier of the `Reservation`. |
```
